lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -373,13 +373,6 @@ class GatewayProvider(
         return getBlockTransactionCount(payload)
     }
 
-    // Copied values from TransactionType in cairo-lang
```

Comment:
Why did you remove it? I don't want to repeat the same magic string multiple times.

Reply:
Other parts of the json are hardcoded and dependent on the type: it's impossible only the `"type"` would change leaving the other values intact so this function does nothing in terms in refactoring or maintainability. If anything it makes it unclear, what exactly is the json object that is being created.

Reply:
I don't want to repeat magic constants.

Reply:
Let's replace it with some better logic then, after #176 is merged